.github/iSSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+<!-- Add a brief and meaningful description. -->
+
+## Expected Behavior
+<!-- Describe the expected behaviour. -->
+
+## Actual Behavior
+<!-- Describe the current/actual behaviour. -->
+
+## Environment
+
+* Operating system: (E.g RHEL 7.6 )
+* OpenShift version:
+<!-- Run the command `oc version` and add the result here. -->
+* Ansible version:
+<!-- Run the command `ansible --version` and add the result here. -->
+* Project Version/Tag: (E.g release-1.0.1)
```

Comment:
@camilamacedo86 As this is a node.js app, I think asking for the Node.js & NPM version is sufficient here.

Reply:
@camilamacedo86 Apologies, I was thinking of a different repo.
THis is a Java project, so Java & Maven should be fine here